### DISCUSSION TITLE
Hardhat how always following the mainnet node states?

### DISCUSSION THREAD

**Discussion by TangMonk** [NomicFoundation/hardhat#3643] [General]
I wanna to create an application for simulate transacation with Hardhat networking forking feature (check the ERC20 token buy and sell tax fee)

I try to config `mining mode` to `auto` or `interval`. But it just mine a fake block (not real sync with mainnet), So I cannot get real data from mainnet.

In order to always sync with mainnet, I have to restart hardhat networking forking every `12s`(the normal block generation time in ethereum). It is painful.

Thanks

**Comment by fvictorio** [comment]
I think [this plugin](https://www.npmjs.com/package/hardhat-live-fork) attempts to do that, but I haven't tried it yet.

**Reply by TangMonk** [reply]
thanks
